tests/test_preprocess.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import sys
+import os
+sys.path.append("..") #Add top level directory to python path for imports
+from corn_app import preprocess
+from PIL import Image
+
+class TestKernelCounter(unittest.TestCase):
+    """ Tests preprocess.py functions
+    """
+
+    def test_preprocess_standardize(self):
+        with open('images/ear-7-512.jpg', 'rb') as image_file:
+            with Image.open(image_file) as image:
+
+                test_function = preprocess.standardize
+
+                self.assertEqual(False, test_function(None))
+                self.assertEqual(True, test_function(image).width == 1280 and test_function(image).height == 720)
```

Comment:
You should import ```STANDARD_WIDTH``` and ```STANDARD_HEIGHT``` here.
You're also resizing the same image object twice with the line `test_function(image).width == 1280 and test_function(image).height == 720`. Just call the function once on the image object before the assertEqual and check the ```image.width``` and ```image.height```.